wofs/wofs_app.py

```diff
@@ -502,7 +529,11 @@ def submit(index: Index,
            queue: str,
            no_qsub: bool,
            time_range: Tuple[datetime, datetime],
-           tag: str):
+           tag: str,
+           dry_run: bool):
+    """
+    Kick off two stage PBS job
+    """
```

Comment:
This needs more explanation about `--dry-run`. Does it still submit another PBS job? Does it create the WOfS output files but not record anything in the Database. How far does it run?

Reply:
Added more explanation to `datacube-wofs submit` `doc string`.